api/tests/unit/features/test_unit_features_views.py

```diff
@@ -2219,6 +2219,42 @@ def test_cannot_update_feature_of_a_feature_state(
     )
 
 
+@pytest.mark.parametrize(
+    "client",
+    [lazy_fixture("admin_master_api_key_client"), lazy_fixture("admin_client")],
+)
+def test_create_feature_with_required_metadata_returns_201(
```

Comment:
What about other tests? e.g. trying to create a feature without the required metadata?